streams-contrib/streams-provider-twitter/src/main/java/org/apache/streams/twitter/provider/TwitterFollowingProviderTask.java

```diff
@@ -48,23 +48,26 @@
     protected TwitterFollowingProvider provider;
     protected Twitter client;
     protected Long id;
+    protected Boolean ids_only;
```

Comment:
chose ids_only as a name to match the corresponding json parameter.  but idsOnly is actually a closer match the configuration bean, so i'll change to that.